main.tf

```diff
@@ -1,18 +1,16 @@
 resource "aws_ram_resource_share" "subnet_sharing" {
-  count                     = length(var.private_subnet_arns) > 0 || length(var.public_subnet_arns) > 0 ? 1 : 0
+  count                     = length(var.private_subnet_arns) >= 0 || length(var.public_subnet_arns) >= 0 ? 1 : 0
```

Comment:
This is a bug which we already discussed last week.. `>=`

Reply:
Added by accident on the MR.